docs/installation.rst

```diff
@@ -17,7 +17,7 @@ After `installing Anaconda <https://docs.anaconda.com/anaconda/install/index.htm
 
     $ conda install -c conda-forge tiatoolbox
 
-Please note that conda-forge installation support is limited on Windows as openslide binaries are not supported on official conda channels. An alternate way to install using conda on Windows could be to install it in `WSL2 with CUDA support <https://docs.microsoft.com/en-us/windows/ai/directml/gpu-cuda-in-wsl>`_.
+Please note that conda-forge installation support is limited on Windows as openslide binaries are not supported on official conda channels. An alternate way to install using conda on Windows could be to install it in `WSL2 with CUDA support <https://docs.microsoft.com/en-us/windows/ai/directml/gpu-cuda-in-wsl>`_. In some cases, WSL2 runs faster on Python codes and therefore we recommend this option.
```

Comment:
Might want to bold the WSL2 bit as it is easy to miss.

Reply:
Addressed in this commit https://github.com/TissueImageAnalytics/tiatoolbox/pull/454/commits/4bfe73945e096dc0b7d07e8d211faeda1289c9a9